internals/secrethub/mkdir.go

```diff
@@ -47,40 +49,41 @@ func (cmd *MkDirCommand) Run() error {
 	}
 
 	for _, path := range cmd.paths {
-		if cmd.parents {
-			err = client.Dirs().CreateAll(path.Value())
-			if err != nil {
-				return err
-			}
+		err := cmd.createDirectory(client, path)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Encountered an error: %s\n", err)
```

Comment:
I'd expect this error to be more in line with the success message.  Now you can get the following output:
```
Encountered an error: Directory or secret already exists, create a different directory (server.dir_already_exists) 
Encountered an error: Directory or secret already exists, create a different directory (server.dir_already_exists) 
Created a new directory at joriscoenen/dirtest/staging/service4
```
This leaves me to guess which directories resulted in an error. 

For example:
```suggestion
fmt.Fprintf(os.Stderr, "Could not create a new directory at %s: %s\n", path, err)
```